src/installAddon.js

```diff
@@ -1,4 +1,4 @@
-import { parseDLURL_curseforge } from './curseforge'
+import { parseDLURLCurseForge } from "./parseDLURLCurseForge";
```

Comment:
Should be 
`import { parseDLURLCurseForge } from "./curseforge";` 

Where `parseDLURLCurseForge` is a function that `installAddon.js` imports from `curseforge.js`